src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -158,6 +169,8 @@ export const EditTxParametersForm = ({
                   min="0"
                   component={TextField}
                   disabled={!areSafeParamsEnabled(parametersStatus)}
+                  onChange={(e: ChangeEvent<HTMLInputElement>) => setManualSafeNonce(e.target.value)}
```

Comment:
I'd move this to a handler instead, e.g. `onSetManualSafeNonce`.

Reply:
I just saw the previous `handleSafeNonceChange`. Did you move it back for a reason?

Reply:
Since its only doing one thing onChange now I moved it back. Are there benefits of having a handler in this case?

Reply:
I think it's easier to follow.